### DISCUSSION TITLE
Incorrect station textDescription

### DISCUSSION THREAD

**Discussion by paulnadler** [weather-gov/api#398] [Q&A]
Many weather stations incorrectly always report "Clear" and the corresponding icon for the fields "textDescription" and "icon".

For example TSTEV
https://api.weather.gov/stations/TSTEV/observations

Comparing to https://www.wrh.noaa.gov/mesowest/getobextXml.php?sid=TSTEV&num=48

Most cases there was no weather condition and in some cases the station reported Lt Rain. The new API always shows clear.

**Comment by scadergit** [comment]
Hi, Paul, just wanted to chime in to let you know we saw this report and we'll investigate. Given priorities it might take a few days.

**Comment by StephenClouse** [comment]
Two things going on here.

Sites showing Clear is a known API issue for observation sites that lack a sky/cloud discriminator. This is common with sites such as road sensors. These will always show clear if there is no present weather to indicate otherwise. As we do not have metadata as to which sites have this equipment, this is a difficult issue to resolve.

However, in this particular case the missing light rain appears to be an upstream data issue. The data we are receiving does not include the present weather for this station. As this feed in part comes from MesoWest, this data should definitely be present as it is appearing on the Western Region site. We are currently working this issue with the data provider.

**Comment by paulnadler** [comment]
Thanks for the update. 

Although less than ideal, I think the approach taken with the old API for this situation is preferable. The website will show no icon and n/a for the summary.

In XML you will see:
```
<name>Weather Type, Coverage, Intensity</name>
<weather-conditions weather-summary="NA"/>
```
and
```
<conditions-icon type="forecast-NWS" time-layout="k-p1h-n1-1">
<name>Conditions Icon</name>
<icon-link>NULL</icon-link>
</conditions-icon>
```

**Comment by scadergit** [comment]
The forecast application is limited to a smaller set of observation stations that could more safely make that assumption, and is more risky with the larger set of stations provided by the API.
